app/main.py

```diff
@@ -1 +1,55 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if not self.is_hungry:
+            return 0
+        print(f"Eating {self.appetite} food points...")
+        self.is_hungry = False
+        return self.appetite
+
+
+class Cat(Animal):
+    def __init__(
+            self,
+            name: str,
+            is_hungry: bool = True
+    ) -> None:
+        super().__init__(name, appetite=3)
```

Comment:
You should pass the `is_hungry` to the `super().__init__(` as well, the user can define this value during the creation of the `Cat` or `Dog` instance. Add it to the `Dog` and `Cat` classes.

Reply:
Hello!
When I add is_hungry to the super().__init__ method in Cat and Dog classes I get a mistake that init takes two positional arguments, but three were giwen.

![image](https://github.com/mate-academy/py-feed-animals/assets/143745512/8531e887-7b14-427d-bbba-8d2e9af81660)

Reply:
In your screenshot, you removed the `is_hungry` from the `__init__` of the `Dog` and `Cat` classes (what is wrong) and added the hardcoded value to the `super().__init__(...` (what is wrong as well). Because of that, you obtain the error. Return the `is_hungry: bool = True` so that you can initialize the `Cat` and `Dog` instances with two arguments as it was before and pass the `is_hungry` value to the  `super().__init__(...` without hardcoding.
```suggestion
        super().__init__(name, appetite=3, is_hungry=is_hungry)
```